src/github.com/stellar/go-horizon/resource_account.go

```diff
@@ -13,11 +16,17 @@ import (
 // AccountResource is the summary of an account
 type AccountResource struct {
 	halgo.Links
-	ID          string            `json:"id"`
-	PagingToken string            `json:"paging_token"`
-	Address     string            `json:"address"`
-	Sequence    int64             `json:"sequence"`
-	Balances    []BalanceResource `json:"balances"`
+	ID            string             `json:"id"`
+	PagingToken   string             `json:"paging_token"`
+	Address       string             `json:"address"`
+	Sequence      int64              `json:"sequence"`
+	Numsubentries int32              `json:"numsubentries"`
+	InflationDest null.String        `json:"inflation_destination"`
+	HomeDomain    null.String        `json:"home_domain"`
+	Thresholds    map[string]byte    `json:"thresholds"`
+	Flags         map[string]bool    `json:"flags"`
```

Comment:
Likewise to the thresholds attribute, this should probably just be a new struct type.